src/UI/Factory.php

```diff
@@ -662,4 +662,36 @@ public function breadcrumbs(array $crumbs);
 	 */
 	public function chart();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A Floatable overlays all other content on the screen. It's free floating
+	 *     nature allows the positioning to some fitting space on the screen. They may
+	 *     persist during navigational changes allowing to cary information through
+	 *     different screens.
+	 *   composition: >
+	 *      Floatables consist of a Title and a glyphed close-button in a header and a
+	 *      content area placed bellow which may offer actions.
```

Comment:
I wonder if that action thingy is an actual good idea. As a user I would somehow expect, that the action is carried out in the Floatable without destroying the page in the background. I also wonder if we want to restrict the actions that could live in a floatable somehow, because floatables most certainly should not become yet another means of random navigation or control for the system, right? The example for the instructional floatable somehow seems to support my case, as there is another location (is there?) where I could submit my portfolio. What are the reasons to put an action in the floatable then? Can we describe these?